docs/RdAnalysis_example.ipynb

```diff
@@ -721,6 +721,58 @@
     "                                         scatter_ymin=0.5, scatter_ymax=1.1,\n",
     "                                         hist_xmin=-30, hist_xmax=45);"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Substituting a different modeled power\n",
+    "Incidentally, you can optionally run the analysis while normalizing with your own calculated expected power.  By default, `RdAnalysis` normalizes by the included `pvwatts_dc_power` calculation.  To override with a different value, pass in `RdAnalysis.power_expected`. You can substitute any modeled power here, like from SAM or PVSyst or PVLib.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "power_expected = rdtools.normalization.pvwatts_dc_power(poa_global=rd.poa,\n",
+    "                                                           power_dc_rated=meta['power_dc_rated'], \n",
+    "                                                           temperature_cell=rd.cell_temperature,\n",
+    "                                                           gamma_pdc=meta['gamma_pdc'])\n",
+    "\n",
+    "rd_with_expected_power = rdtools.RdAnalysis(pv=df['power'], poa=df['poa'],                              \n",
+    "                             power_expected = power_expected,\n",
+    "                             cell_temperature=rd.cell_temperature,\n",
```

Comment:
I think cell temp could be removed here, or is it left in to keep the filtering identical?

Reply:
Yeah, prior to commit 6101b2b it would throw an error, but it can be removed now.  Including cell_temperature would ensure the filtering was identical if there were any unphysical Tcell values (which wasn't the case with this dataset).